appion/protocols/protocol_dogpicker.py

```diff
@@ -36,8 +37,7 @@
 import pyworkflow.utils as pwutils
 from pyworkflow.utils.properties import Message
 
-from convert import readSetOfCoordinates
-
+from ..convert import readSetOfCoordinates
```

Comment:
This will be better: 
from appion.constants import readSetOfCoordinates
Since you are now in a protocol and you should not assume any relative path (unless at the same level), but just that this constant is provided by the 'constants' submodule from appion module (or plugin or Python package)

Reply:
ok